.travis.yml

```diff
@@ -8,8 +8,11 @@ python:
   - 3.7
 
 env:
-  - KARTOTHEK_ARROW_VERSION=0.12.1
   - KARTOTHEK_ARROW_VERSION=0.13.0
+  - KARTOTHEK_ARROW_VERSION=0.12.1
+
+before_cache: rm -f "$HOME/.cache/pip/log/debug.log"
```

Comment:
Is this caching actually required or a nice to have?

Reply:
The build times look worse, this is why I'm asking. This might, of course, be unrelated and simply due to high load on travis but if it's related to this caching, I'd rather not use it

Reply:
a) Good to have
b) I tried a bunch of runs last night - I think there was a difference of ~1 minute; therefore a) good to have. Can be removed.

Reply:
From what I can see the build takes longer than before.

Reply:
Might be because of load; unsure. The time saving I observed at night was ~1 minute so not that big of a difference. Removing.